### ISSUE TITLE
Pytorch Learning Rate should be float

### ISSUE THREAD

**Issue by ADI10HERO** [Auto-DL/Generator#116]
## Bug Report
Not a very important bug as of now

### Current Behavior
Learning rate is int, increases by 1 

### Expected behavior/code
Should be float supporting 10e-9 in the worst case

**Comment by ADI10HERO** [comment]
@rajraj889  Did we solve fix this?

**Comment by rajraj889** [comment]
I guess not i ll check it once though.

**Comment by ADI10HERO** [comment]
Sure, thanks, let me know in any case
